localstack/testing/pytest/fixtures.py

```diff
@@ -669,6 +675,30 @@ def factory(**kwargs):
             LOG.debug(f"error cleaning up route53 HostedZone {zone}: {e}")
 
 
+@pytest.fixture
+def transcribe_create_job(transcribe_client, s3_client, s3_bucket):
+    def _create_job(**kwargs):
+        if "TranscriptionJobName" not in kwargs:
+            kwargs["TranscriptionJobName"] = f"test-transcribe-{short_uid()}"
+
+        if "LanguageCode" not in kwargs:
+            kwargs["LanguageCode"] = "en-GB"
+
+        if "Media" not in kwargs:
+            test_key = "test-clip.wav"
```

Comment:
`test_key` must be defined outside of the `if` block to avoid potential `NameError` on line 693

Reply:
agreed! will push the change..